packages/commands/src/playlist/create-playlist.ts

```diff
@@ -0,0 +1,49 @@
+import { randomBytes } from 'crypto'
+import chalk from 'chalk'
+import { Command } from '@commander-js/extra-typings'
+import { getCurrentUserId, initializeAudiusSdk } from '../utils'
+
+export const createPlaylistCommand = new Command('create')
+  .description('Create playlist')
+  .argument('<trackIds...>', 'Tracks to include in playlist')
+  .option('-f, --from <from>', 'The account to upload from')
+  .option(
+    '-n, --playlist-name <playlistName>',
+    'Name of playlist (chosen randomly if not specified)'
+  )
+  .option(
+    '-d, --description <description>',
+    'Description of playlist (chosen randomly if not specified)'
+  )
+  .option(
+    '-p, --is-private <isPrivate>',
+    'Make playlist private',
+    (arg) => !!arg
```

Comment:
Would this not result in the string value `false` being treated as true?

Reply:
yes, fixing